codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -273,9 +278,10 @@ update these file paths accordingly.
 
     if (!fileWithEntitlements.contains(entitlementCurrentPath) &&
         !fileWithoutEntitlements.contains(entitlementCurrentPath)) {
-      log.severe('The system has detected a binary file at $entitlementCurrentPath.'
-          'but it is not in the entitlements configuartion files you provided.'
-          'if this is a new engine artifact, please add it to one of the entitlements.txt files');
+      log.severe('the binary file $currentFileName is causing an issue');
```

Comment:
Do we need to log out both `currentFileName` AND `entitlementCurrentPath`? Would this be confusing to the person reading these logs?

Reply:
Umm they are both logged currently. It is just that `entitlementCurrentPath` is logged on line 282 and `currentFileName` is logged on line 281. umm maybe we should log them on the same line to be clearer?

Reply:
Changed to logging `entitlementCurrentPath` twice. Open to suggestions on how to make it more readable